simplivity/resources/omnistack_clusters.py

```diff
@@ -116,3 +116,26 @@ def __init__(self, connection, resource_client, data):
         self.data = data
         self._connection = connection
         self._client = resource_client
+        self._clusters = OmnistackClusters(self._connection)
+
+    def __refresh(self):
+        """Updates the omnistack cluster data."""
+        resource_uri = "{}/{}".format(URL, self.data["id"])
+        self.data = self._client.do_get(resource_uri)['omnistack_cluster']
+
+    def set_time_zone(self, time_zone, timeout=-1):
+        """ Sets the time zone for a cluster.
+
+        Args:
+            time_zone: The time zone in case-sensitive region/locale format
+                       for example, "America/New_York"
+            timeout : Time out for the request in seconds.
+        Returns:
```

Comment:
needs a new line in order to the doc parser to parse it correctly

Reply:
Added the new line